core/chains/evm/client/client.go

```diff
@@ -139,8 +126,8 @@ type CallArgs struct {
 	Data hexutil.Bytes  `json:"data"`
 }
 
-// GetERC20Balance returns the balance of the given address for the token contract address.
-func (client *client) GetERC20Balance(ctx context.Context, address common.Address, contractAddress common.Address) (*big.Int, error) {
+// TokenBalance returns the balance of the given address for the token contract address.
+func (client *client) TokenBalance(ctx context.Context, address common.Address, contractAddress common.Address) (*big.Int, error) {
 	result := ""
 	numLinkBigInt := new(big.Int)
 	functionSelector := evmtypes.HexToFunctionSelector("0x70a08231") // balanceOf(address)
```

Comment:
just checking, I know this is not the scope of this PR - will we change these evmtypes here to generic ones & the function selector implementation differences between chains? @aalu1418 @prashantkumar1982